core/src/test/java/org/mskcc/cbio/portal/util/TestCancerStudyReader.java

```diff
@@ -68,8 +68,25 @@ public void testCancerStudyReader() throws Exception {
          cancerStudy = CancerStudyReader.loadCancerStudy( file );
          fail( "Should have thrown DaoException." );
       } catch (Exception e) {
-         assertTrue( e.getMessage().equals( 
-                  "brcaxxx is not a supported cancer type."));
+    	 assertEquals("brcaxxx is not a supported cancer type.", e.getMessage());
       }
    }
+
+	@Test
+	   public void testCancerStudyReaderShortName() throws Exception {
+
+	      File file = new File("target/test-classes/cancer_study.txt");
```

Comment:
Good point. The only need for the `target` version is when Maven filtering is in the way, and it shouldn't be except when we really need it. I'll amend